diff -Nurp pma-original/libraries/classes/Core.php pma/libraries/classes/Core.php
--- pma-original/libraries/classes/Core.php	2018-08-22 15:36:50.000000000 +0200
+++ pma/libraries/classes/Core.php	2018-11-16 15:21:13.670413773 +0100
@@ -278,7 +278,7 @@ class Core
          * Avoid using Response class as config does not have to be loaded yet
          * (this can happen on early fatal error)
          */
-        if (!is_null($GLOBALS['dbi']) && isset($GLOBALS['PMA_Config']) && $GLOBALS['PMA_Config']->get('is_setup') === false && Response::getInstance()->isAjax()) {
+        if (isset($GLOBALS['dbi']) && !is_null($GLOBALS['dbi']) && isset($GLOBALS['PMA_Config']) && $GLOBALS['PMA_Config']->get('is_setup') === false && Response::getInstance()->isAjax()) {
             $response = Response::getInstance();
             $response->setRequestStatus(false);
             $response->addJSON('message', Message::error($error_message));
